src/dmd/target.d

```diff
@@ -26,50 +26,62 @@ import dmd.tokens : TOK;
 import dmd.root.ctfloat;
 import dmd.root.outbuffer;
 
-/***********************************************************
+/**
+ * A data structure that describes a back-end target. At present it is
```

Comment:
s/A data structure that describes/Describes/

I'm a fan of the notion that if words can be removed from a sentence with no loss of meaning, the words should be excised.